var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -190,7 +189,7 @@ def configure_args(self):
         if '+threads' in spec:
             config_args.append('-Dusethreads')
 
-        if spec.satisfies('@5.31'):
+        if spec.satisfies('@5.31,5.33'):
```

Comment:
It might be more robust to check if the number is odd or even. Then this doesn't need to be updated every time.

Reply:
Should be fixed now.